src/main/java/com/joyent/manta/cosbench/MantaStorage.java

```diff
@@ -216,8 +231,17 @@ public void createObject(final String container,
             if (durabilityLevel != null) {
                 headers.setDurabilityLevel(durabilityLevel);
             }
-
-            client.put(path, data, contentLength, headers, null);
+            logger.info("Multipart : /{} {} ", multipart, client.getContext().isClientEncryptionEnabled());
```

Comment:
I think that we don't want to force all users into using multipart. Can you add support to the configuration classes so that you can specify if you want to upload in multipart mode or not?

Reply:
We are not forcing them into using multipart, line 235 checks the multipart flag in the client configuration and if it is set it uses multipart else it does not

Reply:
I do see a `multipart` flag in `CosbenchMantaConfigContext`. Perhaps moving the logging statement into the `this.multipart` if branch might avoid confusion. It's also kinda confusing to output a pair of booleans immediately after a single label, wouldn't this print out as `Multipart : /true false`  for unencrypted MPU?

Reply:
Fair point, the 2 booleans in the statement was something I added because I was having problems with the config working correctly, and I knew that the encryption worked. I will remove the statement.